db/models.py

```diff
@@ -47,8 +47,12 @@ def __str__(self) -> str:
 
 class MovieSession(models.Model):
     show_time = models.DateTimeField()
-    cinema_hall = models.ForeignKey(to=CinemaHall, on_delete=models.CASCADE)
-    movie = models.ForeignKey(to=Movie, on_delete=models.CASCADE)
+    cinema_hall = models.ForeignKey(
+        to=CinemaHall, on_delete=models.CASCADE, related_name="movie_session"
+    )
+    movie = models.ForeignKey(
```

Comment:
The name for backwards relation managers should be plural unless it's One-to-One relation.